File: CodeWars/JavaScript/8Kyu/ObjectOrientedPiracy.js
// URL: https://www.codewars.com/kata/regular-ball-super-ball/train/javascript
class Ship {
  constructor(draft, crew) {
    this.draft = draft;
    this.crew = crew;
  }

  isWorthIt(){
    if ((this.draft - this.crew*1.5) > 20){
      return true;
    }
    else{
      return false;
    }
  }
}

var titanic = new Ship(15, 10);
console.log(titanic.isWorthIt());

File: CodeWars/Python/8Kyu/ObjectOrientedPiracy.py
# URL: https://www.codewars.com/kata/object-oriented-piracy/train/python

class Ship:
  def __init__(self, draft, crew):
    self.draft = draft
    self.crew = crew
  def is_worth_it(self):
    if (self.draft - self.crew*1.5) > 20:
      return True
    else:
      return False

Titanic = Ship(15, 10)
print(Titanic.is_worth_it())
